config/configmaps/install-log-regexes-configmap.yaml

```diff
@@ -31,6 +31,11 @@ data:
       - "LimitExceeded"
       installFailingReason: ResourceLimitExceeded
       installFailingMessage: Resource limit exceeded
+    - name: InvalidSubnetCIDRConfig
+      searchRegexStrings:
+      - "subnet\'s CIDR range"
+      installFailingReason: InvalidSubnetCIDRConfig
+      installFailingMessage: Invalid subnet CIDR provided in install config.
```

Comment:
Slight nit, the regex looks just slightly lacking in an indication of something negative, could we extend a little further to include "is outside of the specified machine networks".

Reply:
Made more specific (assuming we want to go ahead with adding it for now) and renamed to `InvalidInstallConfigSubnet`